examples/kubernetes/Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG BASE_IMAGE_NAME=vault.habana.ai/gaudi-docker/1.16.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.2
+ARG BASE_IMAGE_TAG=latest
+
+FROM ${BASE_IMAGE_NAME}:${BASE_IMAGE_TAG} AS optimum-habana
+
+ARG GAUDI_SW_VER=1.16.1
+ARG OPTIMUM_HABANA_VER=1.12.0
+
+RUN pip install --no-cache-dir optimum-habana==${OPTIMUM_HABANA_VER} && \
+    pip install --no-cache-dir git+https://github.com/HabanaAI/DeepSpeed.git@${GAUDI_SW_VER}
+
+FROM optimum-habana AS optimum-habana-examples
+
+ARG OPTIMUM_HABANA_VER=1.12.0
```

Comment:
```suggestion
```

No need to reintroduce the arg again

Reply:
To use the arg in multiple stages, it needs to be redefined (source: https://docs.docker.com/reference/dockerfile/#scope). However, I can try making it just `ARG OPTIMUM_HABANA_VER` (without the `=1.12.0`)